.typos.toml

```diff
@@ -0,0 +1,29 @@
+# Configuration for https://github.com/crate-ci/typos
+
+[files]
+extend-exclude = [
+    "resources/linux-config-*",
+    "hypervisor/src/kvm/x86_64/mod.rs",
+]
+
+[default.extend-words]
+# External Interrupt
+EXTINT = "EXTINT"
+extint = "extint"
+INOUT = "INOUT"
+Ded = "Ded" # GUIDed
+ba = "ba"
+Cros = "Cros" # CrosVM
+Dake = "Dake"
+Cachable = "Cachable"
```

Comment:
https://github.com/rust-vmm/acpi_tables/issues/20

Reply:
Are we holding this PR for the pending issue from acpi_tables crate?

Reply:
This is fixed now so if you can rebase your version and update.

Reply:
Done